public/client.js

```diff
@@ -0,0 +1,17 @@
+const menuBtn = document.querySelector(".menu-btn");
+const nav = document.querySelector("nav");
+
+menuBtn.addEventListener("click", function () {
+  nav.classList.toggle("show");
+  menuBtn.classList.toggle("move-btn");
+  menuBtn.classList.toggle("cross")
+});
+
+// close menu when clicked outside of it
+document.onclick = function(e){
+    if (!menuBtn.contains(e.target) && !nav.contains(e.target) ) {
```

Comment:
Snap je wat hier staat? Weet je hoe de hoe de _logical AND_ (&&) in JavaScript werkt?